### DISCUSSION TITLE
Activate primary "clipboard" tab after pasting from other tab

### DISCUSSION THREAD

**Discussion by moritzdietz** [hluk/CopyQ#1979] [Q&A] [answered]
I have this workflow of using other tabs which store static content I need to frequently paste.
After this I usually have to use entries from the primary cliboard tab.

I checked the configuration and settings but I couldn't find a built in way to have Copyq switch back to the main tab after pasting from any other tab.

Would this need a global command to have such functionality? Any pointers would be appreciated. :)

**Comment by hluk** [comment, marked as answer]
The following command copies the selected item (on item activation), hides the main window, pastes the clipboard and switches back to the first tab ([here is how to add the command to CopyQ](http://copyq.readthedocs.io/en/latest/faq.html#how-to-load-shared-commands-and-share-them)):

```ini
[Command]
Command="
    copyq:
    select(currentItem())
    hide()
    paste()
    setCurrentTab(tab()[0])"
Icon=\xf362
InMenu=true
Name=Back to Primary Tab
Shortcut=return, enter
```

**Reply by moritzdietz** [reply]
Hey! Thanks for building the command for me. It works perfectly! :)

**Reply by moritzdietz** [reply]
This lead me to finally digging into the Scripting API docs. You've build a tool that I rely on every single day. At work and on my own devices.
I cannot overstate the importance of its existence and the value it provides to me. I cannot give you physical cake, but have a virtual one: 🍰.
